tests/data/reference_graphs/quantized/synthetic_model/EmbeddingBag.dot

```diff
@@ -0,0 +1,5 @@
+strict digraph  {
+"0 /nncf_model_input" [id=0, scope="", type=nncf_model_input];
+"1 TestModel/NNCFEmbeddingBag[_layer]/embedding_bag" [id=1, scope="TestModel/NNCFEmbeddingBag[_layer]", type=embedding_bag];
```

Comment:
This still does not contain the weight quantizer node. Have you registered `EmbeddingBag` in the HW config in any way that I proposed previously?